api/src/main/resources/org/openmrs/api/db/hibernate/OrderGroup.hbm.xml

```diff
@@ -72,6 +72,12 @@
 		    not-null="false" column="parent_order_group" />
 		<many-to-one name="orderGroupReason" class="org.openmrs.Concept"
 		    not-null="false" column="order_group_reason" />
+		 <!-- one-to-many association to all nested orderGroups in this grouping -->
+        <set name="nestedOrderGroups" inverse="true" cascade="delete" order-by="order_group_id"
+            access="field" batch-size="25">
+            <key column="order_group_id" />
```

Comment:
This does not look like that in the commit that i linked in the ticket description.

Reply:
i have fixed it